src/main/java/teammates/logic/api/TaskQueuer.java

```diff
@@ -244,6 +246,21 @@ public void scheduleStudentForSearchIndexing(String courseId, String email) {
                 paramMap, null);
     }
 
+    /**
+     * Schedules to create, update, and delete (C_UD) deadline extension entities, and possibly notify users of them.
+     */
+    public void scheduleDeadlineExtensionsCudAndPossiblyNotify(String courseId, String feedbackSessionName,
```

Comment:
nit: I don't really think PossiblyNotify part is really important, since it's not really the main action and we have the parameter `notifyAboutDeadlines` anyways. I also think Cud could be something else, I think it would be hard to figure out what this is without reading the comment. However, I can't really think of a better name as well. (maybe a generic `scheduleDeadlineExtensionsWorker`?

What do you think?

Reply:
Would "Change" be a good substitute of "Cud"?

I too think `PossiblyNotify` isn't the main part and can be left out from the name:)

Reply:
I was not very sure about the naming for this one. I also wanted to put `scheduleDeadlineExtensionsWorker`, but I decided against it because I felt it does not say anything about what the worker is actually going to do. Though I'm not sure if that's really a problem.

For the "Cud" part, maybe "Changes"? Though I'm not sure if that sounds like it's only about updating. Another idea is "Handling" but that seems as generic as "Worker" that we might as well just use "Worker".

Reply:
"Change" describes "CUD" as oppose to how "View" describes "Read". To me this is ok.